### PR TITLE
Negative durations

### PR DESCRIPTION
**Pull request by datamel**
Fixed bug and close issue #152.
Added bug to check bug was fixed.

### CODE REVIEW

File: metomi/isodatetime/main.py [hunk 1]
```diff
@@ -81,6 +76,10 @@
        The --max=N (default=10) can be used to control the maximum number
        of time points to print in the result.
 
+    Note: Negative durations can be provided in the following ways:
+        \\-PT1H
+        \\\\-PT1H
```
Comment (oliver-sanders) [inline metomi/isodatetime/main.py]: Note to reviewers:

```python
>>> print('\\')
\
```
```

File: metomi/isodatetime/main.py [hunk 2]
```diff
@@ -49,12 +49,7 @@
 
     # 3.  Print ISO8601 duration as total amount of a unit
```
Comment (wxtim) [inline metomi/isodatetime/main.py]: Suggestion
> #3. Print ISO8601 duration as a number of of hours (H), minutes (M) or second (S)

do we need the 3.1 line given that there is only one section here?

Is there a reason that you have got rid fo the examples?

File: metomi/isodatetime/main.py [hunk 3]
```diff
@@ -49,12 +49,7 @@
...
     # 3.1 Into the total number of hours (H), minutes (M) or seconds (S)
-    #     it represents. Note: negative durations should be escaped by telling
-    #     the command to stop processing more options with `--` or by adding a
-    #     backslash in front of the duration.
-    isodatetime --as-total=s PT1H
-    isodatetime --as-total=s -- -PT1H
-    isodatetime --as-total=s \\-PT1H
+    #     it represents.
```
Comment (wxtim) [inline metomi/isodatetime/main.py]: I don't think that there is a test for this functionality - but that's out of scope for this issue.

**Comment by wxtim** [comment]
Happy. 👍

File: metomi/isodatetime/main.py (hunk 3 above)
Comment (datamel) [reply metomi/isodatetime/main.py]: Thanks Tim, Ive accepted those changes and put back the example that wasn't a negative duration - the negative durations have been added a bit further down. 
I added a test for checking the negative durations and these are all passing :)

**Pull request merged**